migrations/1620954224627-add-fee-priority.js

```diff
@@ -0,0 +1,19 @@
+const { saveConfig, loadLatest } = require('../lib/new-settings-loader')
+
+exports.up = function (next) {
+  const newConfig = {
+    wallets_BTC_feeDiscount: '1'
```

Comment:
Can we rename this to feeMultiplier? Also what happens when we need to add discount into another coin? 

It will require a new migration and a new coin in that big if, let's generalize this. It should add the fee multiplier to every configured coin and automatically set that to 1 every time the operator adds a new coin.

Reply:
Hmm, didn't know that this would work for other coins too. The card is very specific about BTC. But I think I understand what you are saying, if in the future we want to add this feat to another coin, the config is already ready for it.